tensorflow/lite/micro/kernels/ethos_u/ethosu.cc

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+/* Copyright 2020-2021 The TensorFlow Authors. All Rights Reserved.
```

Comment:
update to only have 2021.

you can set up a pre-push hook for code style with:
```
cp tensorflow/lite/micro/tools/dev_setup/pre-push.tflm .git/hooks/pre-push
```

Reply:
Fixed